js/apps/admin-ui/src/user/CreateUser.tsx

```diff
@@ -58,24 +81,26 @@ export default function CreateUser() {
     }
   };
 
+  if (!realm) {
+    return <KeycloakSpinner />;
+  }
+
   return (
     <>
       <ViewHeader
         titleKey={t("createUser")}
         className="kc-username-view-header"
       />
       <PageSection variant="light" className="pf-u-p-0">
-        <UserProfileProvider>
-          <FormProvider {...userForm}>
-            <PageSection variant="light">
-              <UserForm
-                realm={realmRepresentation}
-                onGroupsUpdate={setAddedGroups}
-                save={save}
-              />
-            </PageSection>
-          </FormProvider>
-        </UserProfileProvider>
+        <PageSection variant="light">
```

Comment:
Just noticed we have a nested `PageSection` here, I wonder if we really need it.

Reply:
This was redundant, so I've removed it.